x-pack/plugins/ml/public/application/trained_models/models_management/force_stop_dialog.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiConfirmModal } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import type { OverlayStart } from 'kibana/public';
+import type { ModelItem } from './models_list';
+import { toMountPoint } from '../../../../../../../src/plugins/kibana_react/public';
+
+interface ForceStopModelConfirmDialogProps {
+  model: ModelItem;
+  onCancel: () => void;
+  onConfirm: () => void;
+}
+
+export const ForceStopModelConfirmDialog: FC<ForceStopModelConfirmDialogProps> = ({
+  model,
+  onConfirm,
+  onCancel,
+}) => {
+  return (
+    <EuiConfirmModal
+      title={
+        <FormattedMessage
+          id="xpack.ml.trainedModels.modelsList.forceStopDialog.title"
+          defaultMessage="Force stop model {modelId}?"
+          values={{ modelId: model.model_id }}
+        />
+      }
+      onCancel={onCancel}
+      onConfirm={onConfirm}
+      cancelButtonText={
+        <FormattedMessage
+          id="xpack.ml.trainedModels.modelsList.forceStopDialog.cancelText"
+          defaultMessage="Cancel"
+        />
+      }
+      confirmButtonText={
+        <FormattedMessage
+          id="xpack.ml.trainedModels.modelsList.forceStopDialog.confirmText"
+          defaultMessage="Stop"
+        />
+      }
+      buttonColor="danger"
+    >
+      <FormattedMessage
+        id="xpack.ml.trainedModels.modelsList.forceStopDialog.pipelinesWarning"
+        defaultMessage="Selected model has associated pipelines: "
```

Comment:
Maybe reword to something like `The selected model is being used by the following pipelines:` @lcawl any thoughts on the text in this dialog?

Reply:
IMO the impact is unclear. Will the ingest pipelines fail until you start a new deployment or will the inference processor just be skipped? Perhaps something like this:

> You can't use these ingest pipelines until you re-start the model:

Reply:
Changed in https://github.com/elastic/kibana/pull/118563/commits/9ed443322fc32bfb0b8d136bb257ab2f9a60801f

Reply:
Any index request hitting a pipeline that refers to a model that doesn't have a started deployment will fail. The bulk response will contain an item for each one of them explaining the model wasn't deployed.

Reply:
Thanks!  I've added similar information in the docs and API spec via https://github.com/elastic/elasticsearch/pull/81026 and https://github.com/elastic/elasticsearch-specification/pull/1061